docs/source/operations/projections/merc.rst

```diff
@@ -93,62 +95,69 @@ Forward projection
 
 .. math::
 
-    x = k_0 \lambda
+    x = k_0R \lambda; \qquad y = k_0R \psi
 
 .. math::
 
-    y = k_0 \ln \left[ \tan \left(\frac{\pi}{4} + \frac{\phi}{2} \right) \right]
+    \psi &= \log \tan \biggl(\frac{\pi}{4} + \frac{\phi}{2} \biggr)\\
```

Comment:
Regarding \ln vs \log , I personaly liked the former as being non-ambiguous, but this is apparently debated (cf https://mathoverflow.net/questions/279222/logx-or-lnx-to-denote-the-natural-logarithm-in-research-papers ) . If we want to align on \log, similar instances in tobmerc.rst , webmerc.rst and mill.rst should likely be changed too for consistency

Reply:
My slight preference would be for `log`.  But consistency is more important.  So I've reverted to `ln`.